src/main/steps/urls.ts

```diff
@@ -55,6 +55,17 @@ export const CHILDREN_SEX_AT_BIRTH: PageLink = `${CHILDREN}/sex-at-birth`;
 export const CHILDREN_NATIONALITY: PageLink = `${CHILDREN}/nationality`;
 export const CHILDREN_FULL_NAME_AFTER_ADOPTION: PageLink = `${CHILDREN}/full-name-after-adoption`;
 
+export const SIBLING: PageLink = '/sibling';
+export const SIBLING_EXISTS: PageLink = `${SIBLING}/exists`;
+export const SIBLING_COURT_ORDER_EXISTS: PageLink = `${SIBLING}/court-order-exists`;
+export const SIBLING_NAME: PageLink = `${SIBLING}/name`;
+export const SIBLING_ORDER_TYPE: PageLink = `${SIBLING}/order-type`;
+export const SIBLING_ORDER_CASE_NUMBER: PageLink = `${SIBLING}/case-number`;
+export const SIBLING_ORDER_COURT: PageLink = `${SIBLING}/order-court`;
```

Comment:
URLs are visible to users hence it must be clear 
below URLs can be updated to
```suggestion
export const SIBLING_ORDER_TYPE: PageLink = `${SIBLING}/placement-order-type`;
export const SIBLING_ORDER_CASE_NUMBER: PageLink = `${SIBLING}/placement-order-number`;
export const SIBLING_ORDER_COURT: PageLink = `${SIBLING}/placement-order-court`;
```